recipes/psychopy/meta.yaml

```diff
@@ -0,0 +1,108 @@
+{% set name = "psychopy" %}
+{% set pypi_name = "PsychoPy" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.zip
+  sha256: 5df7c0ff494368fcc4ade820d5b83135f82a44740868ed199d1023e181c30f95
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+  entry_points:
+    - psychopy = psychopy.app.psychopyApp:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - {{ cdt('mesa-libgl') }}        # [linux]
+  run:
+    - python >=3.6
+    - python.app  # [osx]
+    - arabic_reshaper
+    - astunparse
+    - cffi
+    - configobj
+    - esprima-python
+    - freetype-py
+    - future
+    - gevent
+    - gitpython
+    - greenlet
+    - hdf5
+    - imageio >=2.5
+    - imageio-ffmpeg
+    - json_tricks
+    - matplotlib
+    - moviepy
+    - msgpack-numpy
+    - msgpack-python
+    - numpy
+    - opencv
+    - openpyxl
+    - pandas
+    - pillow
+    - psutil
+    - pyglet >=1.3
+    - pyobjc-core  # [osx]
+    - pyobjc-framework-quartz  # [osx]
+    - pyopengl
+    - pyosf
+    - pyqmix >=2018.12.13  # [not linux]
+    - pyqt
+    - pyserial
+    - pysoundfile  # [not win]
+    - pytables
+    - python-bidi
+    - python-gitlab
+    - python-sounddevice
+    - pywin32  # [win]
+    - pyWinhook  # [win]
+    - pyyaml
+    - pyzmq
+    - requests
+    - scipy
+    - wxpython !=4.0.2, !=4.0.3
+    - xlrd
+
+test:
+  imports:
+    - psychopy
+    - psychopy.app
+    - psychopy.data
+    - psychopy.event
+    - psychopy.experiment
+    - psychopy.gui
+    - psychopy.hardware
+    - psychopy.monitors
+    - psychopy.preferences
+    - psychopy.sound  # [not win]
+    - psychopy.visual
+  commands:
+    - "psychopy --version"
+
+about:
+  home: https://www.psychopy.org
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: 'Psychology experiment software in Python'
+  description: |
+    PsychoPy is an open-source package for creating psychology stimuli in
+    Python (A real and free alternative to Matlab). PsychoPy combines the
+    graphical strengths of OpenGL with the easy Python syntax to give
+    psychophysics a free and simple stimulus presentation and control package.
+    an optional C extension for a serious speed boost.
+  doc_url:  https://www.psychopy.org/documentation.html
+  dev_url: https://github.com/psychopy/psychopy
+
+extra:
+  recipe-maintainers:
+    - kastman
+    - peircej
```

Comment:
@kastman @peircej Are you happy to be listed as maintainers?

Reply:
@chrisburr Yes, thanks!